### PR TITLE
Task-005

### PR DESCRIPTION
**Pull request by viktor-debre**
Created connection to database in project. 
Installed Pomelo.EntityFramework.MySql by NuGet.
Create simple form for adding and viewing users.

### CODE REVIEW

File: Controllers/HomeController.cs [hunk 1]
```diff
@@ -6,22 +6,41 @@
...
+using Microsoft.EntityFrameworkCore;
 
 namespace LawyerTimeTracker.Controllers
 {
     public class HomeController : Controller
     {
         private readonly ILogger<HomeController> _logger;
-
-        public HomeController(ILogger<HomeController> logger)
+        private ApplicationContext db;
```
Comment (KapNelson) [inline Controllers/HomeController.cs:16]: Explain why you chose this particular variable name? I think it is not entirely correct, in terms of compliance with the real purpose.

**Review by KapNelson — CHANGES_REQUESTED** [review]
Reply to all comments

File: Controllers/HomeController.cs (hunk 1 above)
Comment (viktor-debre) [reply Controllers/HomeController.cs]: Thank you for the advice, this name is really inappropriate in this situation. It has been already changed to databaseContext, which is easier to understand.

File: appsettings.Development.json [hunk 2]
```diff
@@ -1,4 +1,7 @@
 {
+  "ConnectionStrings": {
+    "MariaDbConnectionString": "server=localhost;port=3306;user id=root;password=tatoserg4;database=lawyer_db"
```
[the remark this answers is not in the document]
Comment (viktor-debre) [reply appsettings.Development.json]: Added as another task on canbas board.

**Pull request merged**